test/index.js

```diff
@@ -564,6 +565,12 @@ describe('maybe', function() {
                     errorEq(Error, 'Cannot instantiate Maybe'));
     });
 
+    it('has its type definition exported', function() {
+      eq(S.MaybeType($.Number).test(S.Nothing()), true);
+      eq(S.MaybeType($.Number).test(S.Just(1)), true);
+      eq(S.MaybeType($.Number).test('abc'), false);
+    });
+
```

Comment:
This could live in its own `describe` block (before the `describe('Maybe')` block, perhaps).